index.html

```diff
@@ -131,6 +131,32 @@ <h2 class="connect">LET'S CONNECT</h2>
             </ul>
             </div>
         </section>
+        <Section class="form-Container">
+            <form id="form" action="https://formspree.io/f/mwkyoojy" method="post">
+                <h2 id="contact-Me">Contact Me</h2>
+                <p class="contact-Me-Paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+                <ul class="form-Ul">
+                    <li class="form-Li">
+                      <label for="name"></label>
+                      <input type="text" id="name" name="user_name" placeholder="Full Name" required>
+                    </li>
+                    <li class="form-Li">
+                      <label for="mail"></label>
+                      <input type="email" id="mail" name="user_email" placeholder="example@gmail.com" required>
+                    </li>
+                    <li class="form-Li">
+                      <label for="msg"></label>
+                      <textarea id="msg" name="user_message" placeholder="Write your message here..." required></textarea>
+                    </li>
+                    <li class="button">
+                        <button id="last-Button" type="submit">Get in Touch</button>
```

Comment:
- [ ] Please make sure your design follows the original [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design guidelines(fonts, colors, margins, paddings etc)
- [ ] If you compare your input and button designs to the original you would see the difference. Please ensure to make it look like the original design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/154321389-b1fdcd3f-f261-4f2c-8135-5bb5adfafcc8.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/154321219-496bb957-ca49-403c-a11d-131f6f9c7413.png)

Reply:
added